native/packages/react-native-bpk-touchable-overlay/src/BpkTouchableOverlay.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, ViewPropTypes, TouchableWithoutFeedback, Platform, StyleSheet } from 'react-native';
+
+const tokens = Platform.select({
+  ios: () => require('bpk-tokens/tokens/ios/base.react.native.common.js'), // eslint-disable-line global-require
+  android: () => require('bpk-tokens/tokens/android/base.react.native.common.js'), // eslint-disable-line global-require
+})();
+
+const styles = StyleSheet.create({
+  overlay: {
+    flex: 1,
+    position: 'absolute',
+    top: 0,
+    right: 0,
+    bottom: 0,
+    left: 0,
+    backgroundColor: tokens.colorGray900,
+    opacity: 0,
+  },
```

Comment:
This is pedantic, but don't we usually list style props in alphabetical order?

Reply:
Erm not that I am aware of - we should set a convention though. In Sass we have a specific order we enforce via stylelint. We should have something similar for "css in js". Ill create a work item.